luarules/gadgets/unit_shield_behaviour.lua

```diff
@@ -182,6 +246,22 @@ local function shieldNegatesDamageCheck(unitID, unitTeam, attackerID, attackerTe
 	return false
 end
 
+local function setProjectilesAlreadyInsideShield(shieldUnitID, radius)
+	-- This section is to allow slower moving projectiles already inside the shield when it comes back online to damage units within the radius.
+	local x, y, z = spGetUnitPosition(shieldUnitID)
+	-- Engine has GetProjectilesInRectangle, but not GetProjectilesInCircle, so we have to square the circle
+	-- TODO: Change to GetProjectilesInCircle once it is added
+	local radius = radius * math.sqrt(math.pi) / 2
```

Comment:
Looks suspicious, why sqrt(pi)? I know it's just copying existing (and presumably working good enough) code but perhaps it's an opportunity to look at.

Reply:
gonna be honest, I'm not exactly sure what's going on here. Watch the Fort squared my circle for me when the shield turns back on and there's projectiles within its radius.  Copilot seems to think it works great, and it works in game. But copilot tends to think almost anything complicated is great.

Reply:
@WatchTheFort